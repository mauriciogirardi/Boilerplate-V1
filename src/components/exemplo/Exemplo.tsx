export function Exemplo() {
  return (
    <div>
      <h2>Mauricio</h2>
    </div>
  )
}
