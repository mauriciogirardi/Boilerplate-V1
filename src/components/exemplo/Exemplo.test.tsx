import { render, screen } from '@testing-library/react'
import { Exemplo } from './Exemplo'

describe('<Exemplo/>', () => {
  it('should render correctly', () => {
    const { container } = render(<Exemplo />)

    expect(
      screen.getByRole('heading', { name: /Mauricio/i })
    ).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })
})
