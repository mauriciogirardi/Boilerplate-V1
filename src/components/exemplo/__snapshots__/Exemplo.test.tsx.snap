// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Exemplo/> should render correctly 1`] = `
<div>
  <h2>
    Mauricio
  </h2>
</div>
`;
